haystack/modeling/model/language_model.py

```diff
@@ -512,7 +538,7 @@ def forward(
             attention_mask=padding_mask,
             output_hidden_states=output_hidden_states,
             output_attentions=output_attentions,
-            return_dict=False
+            return_dict=False,
```

Comment:
is this trailing comma black convention? why?

Reply:
Yep, trailing commas are a convention of Black: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas 

AFAIK is used to make every line always end in a comma, regardless of its position in the list/signature, which in general is helpful if you later on add other elements and has no impact otherwise.